package-lock.json

```diff
@@ -32,6 +32,23 @@
         }
       }
     },
+    "@babel/runtime": {
```

Comment:
Please remove changes to `package-lock.json`. They are hard to review from a security perspective. I will rebuild `package-lock.json` when your PR is merged,